src/components/DataView/DataViewActionsRow/DataViewActionsRow.tsx

```diff
@@ -7,7 +7,7 @@ import DataViewPager from "../DataViewPager";
 import { DataViewActionsRowWrapper, LeftControlsContainer, RightControlsContainer } from "./DataViewActionsRow.styled";
 import { DataViewActionsRowProps } from "./DataViewActionsRowTypes";
 import Checkbox from "../../Checkbox";
-import DataViewBulkActionsButtonsRow from "../DataViewBulkActionsButtonsRow";
+import DataViewBulkActionsButtonsRow from "../DataViewBulkActionsButtonsRow/DataViewBulkActionsButtonsRow";
```

Comment:
I think this import can be done:
`import DataViewBulkActionsButtonsRow from "../DataViewBulkActionsButtonsRow";`
Since the `DataViewBulkActionsButtonsRow` has an index file.